modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -119,6 +119,9 @@ else
   gsutil -m rsync -d -r gs://${storage_bucket_name}/policy-library ${policy_library_home}/policy-library || echo "No policy available, continuing with Forseti installation"
 fi
 
+# Attempt to download the Forseti scripts and gracefully handle the absence of scripts.
+gsutil -m cp -r gs://${storage_bucket_name}/scripts ${forseti_scripts}/ || echo "No scripts available, continuing with Forseti installation."
```

Comment:
If there are no scripts, maybe we should throw an error. Without the run_server and init scripts, Forseti will not work.

Reply:
Removed message.